symphony-ws/src/main/java/se/havochvatten/symphony/scenario/ScenarioService.java

```diff
@@ -178,7 +178,7 @@ GridCoverage2D apply(GridCoverage2D coverage,
                     state,
                     (GridCoverage2D innerState, BandChange bandChange) -> {
                         LOG.info("Applying changes for feature {}: " + bandChange,
-                            area.getFeature().getProperty("name").getValue().toString());
+                            area.getName());
```

Comment:
Nice simplification!  👍

Reply:
:relaxed: It serves to avoid some code repetition as well.

Reply:
I'm sure you noticed that I made it into a method on the entity:

https://github.com/havochvatten/MSP-Symphony/blob/e4431e670ddb3beae746f9e1b2f615f73398d76a/symphony-ws/src/main/java/se/havochvatten/symphony/scenario/ScenarioArea.java#L203-L206